src/lib/pk_pad/eme_oaep/oaep.cpp

```diff
@@ -91,10 +91,10 @@ secure_vector<byte> OAEP::unpad(const byte in[], size_t in_length,
 
    mgf1_mask(*m_hash,
              &input[m_Phash.size()], input.size() - m_Phash.size(),
```

Comment:
No. It would be if m_Phash would be empty, but this can be ruled out (see constructor).